.github/workflows/ci-lite-test-full.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       id: skip
       shell: bash -l {0}
       run: |
-        FILTER='src/pytorch_lightning|requirements/pytorch|tests/tests_pytorch|examples/pl_*'
+        FILTER='src/lightning_lite|requirements/lite|tests/tests_lite'
```

Comment:
Should this be removed too @carmocca ?

Reply:
Done in https://github.com/Lightning-AI/lightning/pull/14451/commits/fd6df9bb9f93509a788a0a53f64dabde5e6c9b97